pretext/pretext.py

```diff
@@ -1479,7 +1480,7 @@ def webwork_to_xml(
             if ww_image_scheme:
                 image_url = ww_image_url
             else:
-                image_url = ww_domain + "/" + ww_image_full_path
+                image_url = urllib.parse.urljoin(ww_domain,ww_image_full_path)
```

Comment:
Mind if I nitpick?

```suggestion
                image_url = urllib.parse.urljoin(ww_domain, ww_image_full_path)
```

You should be able to just click something in GitHub to add this to your branch. Or you could make the change locally and push.

Reply:
Nitpicks welcome! Change on its way.